package.json

```diff
@@ -25,6 +25,7 @@
     "framer-motion": "^10.12.10",
     "next": "13.3.2",
     "node-fetch": "^3.3.1",
+    "open-graph-scraper": "5.2.3",
```

Comment:
If its version is the latest version `6.0.1', an error occurs in Vercel: https://vercel.com/de-store/frontend-shop-dao-generator-chainlink2023/3BgBb7A39pKUyZ8ucfifHdth67uu
So, I set '5.2.3' temporarily